spec/models/abilities/image_abilities_spec.rb

```diff
@@ -54,6 +54,36 @@
     end
   end
 
+  context "As a Charity user" do
```

Comment:
I think this will be for all users not only Charity user so we might want to change description of spec. 🤔 💭